sujeong/app.js

```diff
@@ -45,12 +47,137 @@ app.post('/users',async (req ,res,next)=>{
             email,
             profile_image,
             password
-        ) VALUES ( ?, ?, ?, ?)
+        ) VALUES ( ?, ?, ?, ?);
         `,[name, email, profile_image, password]
     );
     res.status(201).json({message:"userCreated"})
 })
 
+
+app.post('/posts',async(req, res, next)=>{
+    const {title, content, user_id,image_url} =req.body
+    
+    await appDataSource.query(
+        `INSERT INTO posts(
+            title,
+            content,
+            user_id,
+            image_url
+        ) VALUES (?,?,?,?);`
+    ,[title, content, user_id, image_url]
+    );
+    res.status(201).json({message: "postCreated"})
+})
+
+
+app.get('/posts/userId', async(req, res)=>{
+    
+    await appDataSource.query(
+    `SELECT
+        u.id as userId,
+        u.profile_image as userProfileImage,
+        p.user_id as postingId,
+        p.image_url as postingImageUrl,
+        p.content as postingContent
+    FROM users u
+    INNER JOIN posts p ON u.id = p.user_id;
+    `,(err, rows) => {
+        res.status(200).json(rows);
+    });
+    });
```

Comment:
수정님 ! 해당 스코프에 문제가 없는 지 다시 확인 해주세요 ~!
```suggestion
    await appDataSource.query(
    `SELECT
        u.id as userId,
        u.profile_image as userProfileImage,
        p.user_id as postingId,
        p.image_url as postingImageUrl,
        p.content as postingContent
    FROM users u
    INNER JOIN posts p ON u.id = p.user_id;
    `,(err, rows) => {
        res.status(200).json(rows);
    });
```

Reply:
네 잘 되요~